docker_images/fastai/requirements.txt

```diff
@@ -0,0 +1,4 @@
+starlette==0.14.2
+api-inference-community==0.0.23
+fastai
+git+https://github.com/huggingface/huggingface_hub
```

Comment:
We need the function`from_pretrained_fastai` which is just being added in the V0.6.0 of `huggingface_hub` ([link](https://github.com/huggingface/huggingface_hub/releases)). 

I added to the requirements `huggingface_hub==0.6.0rc0` since it is currently in pre-release. I can update this as soon we have the official release.